Containers/imaginary/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:latest
 FROM golang:1.22.2-alpine3.18 as go
 
-ENV IMAGINARY_HASH 6cd9edd1d3fb151eb773c14552886e4fc8e50138
+ENV IMAGINARY_HASH 6cd9edd1d3fb151eb773c14552886e4fc8e50138
 
 RUN set -ex; \
     apk add --no-cache \

```

Comment:
here as well or rather not?

Reply:
The preinstalled packages aren't statucally linked during build, so I think it isn't needed

Reply:
ok